### PR TITLE
Happy new year! 🎉🎉🎉

### PR DESCRIPTION
**Pull request by weiqingtoh**

### CODE REVIEW

**Comment by allenwq** [comment]
@weiqingtoh How about use `Time.now.year` instead, so that we don't need to update every year.

**Comment by weiqingtoh** [comment]
Also can, will there be a function call on page load though?

**Comment by allenwq** [comment]
The function call time is small enough to be ignored, there could be thousands of calls to render this page.

**Comment by weiqingtoh** [comment]
@allenwq fixed.

File: views/layouts/coursemology.org.html.slim [hunk 1]
```diff
@@ -109,4 +109,4 @@ html
                   = link_to('Contact Us', page_path('contact_us'))
                 li.footer-link
                   a href='https://github.com/Coursemology' target='_blank' GitHub
-            p © 2013 - 2016 Coursemology.org. All Rights Reserved.
+            p © 2013 - #{Time.zone.now} Coursemology.org. All Rights Reserved.
```
Comment (allenwq) [inline views/layouts/coursemology.org.html.slim]: `.year` ?

**Comment by allenwq** [comment]
Thanks @weiqingtoh !

**Pull request merged**